apps/meeting/scripts/update-react-library.js

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env node
+
+/**
+ * This script clones and installs the latest code from amazon-chime-sdk-component-library-react
+ * GitHub repository.
+ */
+
+const fs = require('fs');
+const path = require('path');
+const { logger, spawnOrFail } = require('./utilities');
+const rimraf = require('rimraf');
+const packageJSON = require('../package.json');
+
+const dir = 'updated-react-library';
```

Comment:
Can we call it what it is `amazon-chime-sdk-component-library-react-github`?

Reply:
or github packages if we want to maybe do js sdk in the future :D

Reply:
Yeah makes sense, will update it thanks!